How-To/Use-the-Data-team-server.md

```diff
@@ -0,0 +1,63 @@
+# How to use the Data team server
+
+## Contents
```

Comment:
The one section that was suggested in the issue (#5) but that I left off here is a "Resources" section. I couldn't really think of any resources that would be useful above and beyond the [Linux and devops section of our handbook](https://github.com/ccao-data/wiki/blob/jeancochrane/create-explainer-for-data-server/Handbook/Resources.md#git-linux-and-devops), but I'm down to add a section to this doc if you have resources in mind.

Reply:
I agree, let's just add any further resources here to the handbook.